pandas/core/indexers.py

```diff
@@ -376,6 +376,27 @@ def unpack_1tuple(tup):
     return tup
 
 
+def check_key_length(columns, key, value):
+    """
+    Checks if a key used as indexer has the same length as the columns it is
+    associated with.
+
+    Parameters
+    ----------
+    columns: Index The columns of the DataFrame to index.
```

Comment:
space between "columns" and colon, same on the next two lines

Reply:
Thx